docs/eloquent-models.txt

```diff
@@ -0,0 +1,522 @@
+.. _laravel-eloquent-models:
+
+===============
+Eloquent Models
+===============
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: php framework, odm, code example
+
+This package includes a MongoDB enabled Eloquent class that you can use to
+define models for corresponding collections.
+
+Extending the base model
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To get started, create a new model class in your ``app\Models\\`` directory.
```

Comment:
This double backslash is displayed: https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/#extending-the-base-model

Reply:
> This double backslash is displayed: https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/eloquent-models/#extending-the-base-model

Thanks for catching that, and sorry I missed it.
I will revert this and the one in Upgrading and double-check on staging before updating this PR.

Reply:
<img width="662" alt="Screenshot 2024-01-10 at 1 22 29 PM" src="https://github.com/mongodb/laravel-mongodb/assets/52428683/d04bdc84-8c90-4b3c-a939-7be50d8a38e1">
<img width="588" alt="Screenshot 2024-01-10 at 1 21 40 PM" src="https://github.com/mongodb/laravel-mongodb/assets/52428683/3c8ef707-f1a9-49a1-b15e-6b85f62f82fd">